R/busy-indicators.R

```diff
@@ -117,49 +128,86 @@ useBusyIndicators <- function(..., spinners = TRUE, pulse = TRUE) {
 #' shinyApp(ui, server)
 busyIndicatorOptions <- function(
   ...,
+  spinner_type = NULL,
   spinner_color = NULL,
   spinner_size = NULL,
   spinner_delay = NULL,
-  spinner_selector = NULL,
+  spinner_selector = ":root",
```

Comment:
If `spinner_selector = NULL` (and if pulse options aren't customized in the `busyIndicatorOptions()` call), we can return the spinner CSS variables inline. 

It's a nice way to avoid having to add an extra class if you're customizing the spinner for a card, e.g. as used in the example app in the OP.